report-viewer/src/views/ComparisonView.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <div class="container">
+    <button id="show-button" :class="{hidden : !hideLeftPanel}" @click="togglePanel" title="Show sidebar">
+      <img src="@/assets/double_arrow_black_24dp.svg" alt="show">
+    </button>
+    <div id="sidebar" :class="{ hidden : hideLeftPanel }">
+      <div class="title-section">
+        <h1>JPlag Comparison</h1>
+        <button id="hide-button" @click="togglePanel" title="Hide sidebar">
+          <img src="@/assets/keyboard_double_arrow_left_black_24dp.svg" alt="hide"></button>
+      </div>
+      <TextInformation label="Submission 1" :value="id1" :anonymous="store.state.anonymous.has(id1)"/>
+      <TextInformation label="Submission 2" :value="id2" :anonymous="store.state.anonymous.has(id2)"/>
+      <TextInformation label="Match %" :value="comparison.match_percentage"/>
+      <MatchTable :matches="comparison.allMatches" :id1="id1" :id2="id2" @match-selected="showMatch" />
+    </div>
+    <FilesContainer :matches="comparison.matchesInFirstSubmission" :files="filesOfFirst" :container-id="1"
+                    files-owner="Submission 1"
+                    @toggle-collapse="toggleCollapseFirst"
+                    @line-selected="showMatchInSecond"/>
+    <FilesContainer :matches="comparison.matchesInSecondSubmissions" :files="filesOfSecond" :container-id="2"
+                    files-owner="Submission 2"
+                    @toggle-collapse="toggleCollapseSecond"
+                    @line-selected="showMatchInFirst"/>
+  </div>
+</template>
+
+<script>
+import { defineComponent, ref } from "vue";
```

Comment:
This also looks like a wrong ESLint setting (or ESLint is not running at all). Formatting should be consistent.